cmd/worker/main.go

```diff
@@ -65,23 +71,90 @@ func main() {
 		logger.Info("Configuration file not found.")
 	}
 
+	// Temporal client.
+	tc, err := temporalsdk_client.Dial(temporalsdk_client.Options{
+		HostPort:  cfg.Temporal.Address,
+		Namespace: cfg.Temporal.Namespace,
+		Logger:    temporal.Logger(logger.WithName("temporal")),
+	})
+	if err != nil {
+		logger.Error(err, "Unable to create Temporal client.")
+		os.Exit(1)
+	}
+	defer tc.Close()
```

Comment:
Moving the Temporal client creation outside `cmd.go`, to be able to share it with the API server.